app/routes/lending/components/LendableObjectAdminDetail.tsx

```diff
@@ -0,0 +1,88 @@
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import { LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Helmet } from 'react-helmet-async';
+import { useParams } from 'react-router-dom';
+import { fetchLendableObject } from 'app/actions/LendableObjectActions';
+import { fetchAllLendingRequests } from 'app/actions/LendingRequestActions';
+import { Content } from 'app/components/Content';
+import NavigationTab from 'app/components/NavigationTab';
+import { selectLendableObjectById } from 'app/reducers/lendableObjects';
+import { selectAllLendingRequests } from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import type { Params } from 'react-router-dom';
+
+const LendableObjectAdminDetail = () => {
+  const { lendableObjectId } = useParams<Params>();
```

Comment:
What's `Params` here?

Reply:
Not resolved

Reply:
`useParams()` has a return value of `string | undefined` so all the occurrences where Params is explicitly defined did nothing. Removed them now.

Reply:
Yes but my point was that it's wrong to use the `Params` type here

Reply:
You should still type your params, but using `Param` is not correct. Revert https://github.com/webkom/lego-webapp/pull/4219/commits/f1ff1f1380b38494a98cb15759b3ccb3e68dbcc6

Reply:
Hmm, I now see that it is done everywhere else so I agree with keeping it consistent. But I do not see the purpose of why it is done like this. So it would be like this:

```ts
const { lendableObjectId } = useParams<{ lendableObjectId: string }>();
```

however the type given to lendableObjectId is still `string | undefined`, just like without typing it, so it feels like unnecessary bloat.